### DISCUSSION TITLE
Interface  open-webui in Chinese

### DISCUSSION THREAD

**Discussion by axil76** [open-webui/open-webui#1090] [Q&A] [answered]
Hello,
I have just deployed the open-webui container and it is in Chinese, how do we change the language?
Thanks for your help

**Comment by axil76** [comment]
![2024-03-07 17 22 32](https://github.com/open-webui/open-webui/assets/1433185/903ff6db-dd97-434f-acdf-c68d86796f73)

**Reply by justinh-rahb** [reply]
Open-WebUI doesn't at the moment have translation support merged into the `main` branch, nor has anybody contributed a Chinese translation to the `add-i18n` branch that's being worked on right now, so I don't know how this would be possible. Do you have a browser extension that does translation?

**Reply by axil76** [reply]
![2024-03-07 17 45 18](https://github.com/open-webui/open-webui/assets/1433185/80157888-fa01-4f18-ad87-f511c62f8188)
I don't have a browser extension that does the translation, I tried on Firefox and Edge and I have the same behavior.

**Reply by justinh-rahb** [reply, marked as answer]
This is bizarre.. could you share where you got the container or instructions from? Are you using some other forked container build not from the `open-webui` GHCR repo? That's the only thing I can think of here, there's not a single Chinese pinyin character in our codebase at all.

We only provide and support Docker images here:
```
ghcr.io/open-webui/open-webui:main
```

**Reply by axil76** [reply]
indeed, I had not taken the correct qiruizheng/open-webu image because my proxy could not resolve the ghcr.io registry, I just imported the image manually, on the other hand is it what is it possible to start the image without the application trying to download models (requests.exceptions.ProxyError: (MaxRetryError('HTTPSConnectionPool(host=\'huggingface.co\', port=443))? we can do this by hand afterwards

**Comment by PeterDaveHello** [comment]
Looks like it's added in [v0.1.113](https://github.com/KarlLee830/open-webui/releases/tag/v0.1.113) since https://github.com/open-webui/open-webui/commit/dbdc60279175e7cc2ad6b04ea96480b21ea728e6

Ref: https://github.com/open-webui/open-webui/pull/926
